src/app/router/router.tsx

```diff
@@ -0,0 +1,10 @@
+import { createBrowserRouter } from 'react-router-dom';
+import { RootRoute } from './ui/RootRoute';
+
+export const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <RootRoute />,
+    errorElement: <div>ERROR</div>,
```

Comment:
For error component let's create an additional file 
the structure will be src/app/router/ui/ErrorRoute
and let's add error text inside based on the error, if it's 404, then let's show `Page is not found` or `Sorry, an unexpected error has occurred.` for other cases + let's add translations for these texts

Reply:
https://github.com/simeontg/cinema-frontend/pull/3/commits/d878e8bb84630191bf981962a5198f44d4627aaa#diff-a4033043525fe1fde491528a8d20dc71a7258b7031de09994e95b491f2913483